packages/dds/tree/api-report/tree.legacy.alpha.api.md

```diff
@@ -70,9 +70,6 @@ export interface FieldSchemaUnsafe<out Kind extends FieldKind, out Types extends
     readonly kind: Kind;
 }
 
-// @alpha
-export type FixRecursiveArraySchema<T> = T extends TreeNodeSchema ? undefined : undefined;
```

Comment:
This API is alpha, not alpha+legacy, so it should not be removed from the alpha API report and alpha API surface.

Reply:
Same for getJsonSchema

Reply:
Oh, this diff is for the rename from alpha to legacy, so I guess these should be removed in this diff.

So am I correct that this change is removing the alpha report, so that we only have alpha + legacy, and thus all of the alpha non-legacy APIs will not be made usable by this change? I thought we could use alpha for alpha APIs now, is that not the case? DO we need to make all our alpha APIs beta?

Reply:
tree has no /alpha export. /alpha is not listed in package.json exports. So, any `@alpha` APIs are currently only accessible via /internal. If tree wants to have `@alpha` APIs, then /alpha should be added. Sounds like a follow-up PR.

Reply:
In some sense, this PR is correcting a report lie.

Reply:
When the time comes for tree to have real alpha, this will be the first time a package has both alpha and legacy exports.
See [api-extractor-report.esm.current.json](https://github.com/microsoft/FluidFramework/blob/main/common/build/build-common/api-extractor-report.esm.current.json) with this comment:
```json
	"apiReport": {
		// NOTE: "alpha" is temporarily not included to reduce noise while there are no uses.
		//  Once any package with /legacy also uses /alpha, "alpha" should be added in.
		"reportVariants": ["public", "beta"]
	},
```
probably tree doesn't want to add alpha reports for everything; so, instead it could add:
```
	"apiReport": {
		"reportVariants": ["public", "beta", "alpha"]
	},
```
to its api-extractor/api-extractor.current.json file.